src/array.js

```diff
@@ -49,10 +49,19 @@ TransisArray.of = function() {
 // Public: Creates a new `Transis.Array` from the given array-like object. Useful for converting a
 // regular array to a `Transis.Array`.
 //
-// a - An array-like object.
+// a       - An array-like object.
+// mapFn   - Map function to call on every element of the array (optional).
+// thisArg - Value to use as this when executing mapFn (optional).
 //
 // Returns a new `Transis.Array`.
-TransisArray.from = function(a) { return TransisArray.of.apply(null, a); };
+TransisArray.from = function(a, mapFn, thisArg) {
+  if(mapFn) {
+    for (var i = 0; i < a.length; i++) {
+      a[i] = mapFn.call(thisArg || this, a[i]);
```

Comment:
This is mutating the passed in object, which would be unexpected in this case. You should create a new array to store the mapped values.